setup.py

```diff
@@ -26,10 +26,12 @@
 	long_description_content_type='text/x-rst',
 	install_requires=[
 		'numpy',
-		'scipy',		# for phash
-		'pillow',		# or PIL
-		'PyWavelets',  # for whash
+		'pillow',	# or PIL
 	],
+	extras_require={
+        "whash": "PyWavelets",	# for whash
+		'phash': "scipy",		# for phash
+    },
```

Comment:
You don't **have** to provide these. Users can just add to their own dependencies. But it's a nice to have. And helps some tools that look at packages co-dependencies (for security analysis for instance)